interface/clickmap/template/general_new.html

```diff
@@ -11,60 +11,54 @@
 	</head>
 	<body>
 {/if}
-<div class="outer-container">
-	<div id="container" class="container graphic-pain-map">
-		<img src="{$form->image}"/>
-	</div>
-	<div id="legend" class="legend graphic-pain-map">
-		<div class="body">
-			<ul></ul>
-		</div>
-	</div>
+<div class="container">
+    <div class="row">
+        <div class="col-9">
+            <div id="container" class="container graphic-pain-map">
+                <img src="{$form->image}"/>
+            </div>
+        </div>
+        <div class="col-3">
+            <div id="legend" class="legend graphic-pain-map">
+                <div class="body">
+                    <ul></ul>
+                </div>
+            </div>
+        </div>
+        <div class="col-12">
+            <button class="btn btn-primary btn-save" id="btn_save">{xl t="Save"}</button>
+            <button class="btn btn-secondary btn-delete" id="btn_clear">{xl t="Clear"}</button>
+            <button class="btn btn-danger btn-cancel" id="cancel">{xl t="Cancel"}</button>
```

Comment:
this should be a `btn-secondary`
(rec getting rid of the crazy blue blackground on this form also; note back in the day a lot of OpenEMR looked like that :) )